src/early_multicellular_stage/editor/ReproductionOrderEditor.cs

```diff
@@ -0,0 +1,196 @@
+﻿using System.Collections.Generic;
+using Godot;
+
+/// <summary>
+///   This is used to change the order hexes are created in for hex-based organisms (i.e. <see cref="MicrobeSpecies"/>
+///   and <see cref="EarlyMulticellularSpecies"/>).
+/// </summary>
+public class ReproductionOrderEditor<THex, TEditor, TCombinedAction, TAction, THexMove, TContext> : MarginContainer
+    where THex : IActionHex
+    where TEditor : class, IHexEditor, IEditorWithActions
+    where TCombinedAction : CombinedEditorAction
+    where TAction : EditorAction
+    where THexMove : HexWithData<THex>, IActionHex
+{
+    [Export]
+    public NodePath ReproductionOrderListPath = null!;
```

Comment:
First NodePath in a class needs to be nullable to allow detecting when this object is created in the Godot editor and avoiding error conditions happening there. Editor process created class instances never get their Export variables set and don't have _Ready called on them, but *do* have Dispose called on them.